### ISSUE TITLE
Critical error at startup

### ISSUE THREAD

**Issue by gerard94** [duniter/sakia#740]
Sakia v0.32.2 Ubuntu-gnome 16.10

    An unhandled exception occured

    ----
    Traceback (most recent call last):

  File "sakia/main.py", line 100, in <module>

  File "sakia/app.py", line 90, in startup

  File "sakia/app.py", line 106, in load_profile

  File "sakia/app.py", line 118, in instanciate_services

  File "sakia/data/processors/nodes.py", line 18, in initialize_root_nodes

  File "sakia/data/processors/nodes.py", line 103, in nodes

  File "sakia/data/repositories/nodes.py", line 101, in get_all

  File "sakia/data/repositories/nodes.py", line 101, in <listcomp>

  File "<attrs generated init c39962a7e180e8b055bbbb03cb070bcb31a5b59b>", line 4, in __init__
    self.endpoints = __attr_convert_endpoints(endpoints)

  File "sakia/data/entities/node.py", line 16, in _tuple_of_endpoints

  File "site-packages/duniterpy/documents/peer.py", line 106, in endpoint

  File "site-packages/duniterpy/documents/peer.py", line 133, in from_inline

IndexError: list index out of range

**Comment by Insoleet** [comment]
Duplicate of : https://github.com/duniter/sakia/issues/739

I'd like to get your g1.db file from ~/.config/sakia/Default Profile !

**Comment by Nanocryk** [comment]
I have a similar crash : 
```

    An unhandled exception occured

    ----
    Traceback (most recent call last):

  File "sakia\main.py", line 112, in <module>

  File "site-packages\quamash\__init__.py", line 263, in run_until_complete

RuntimeError: Event loop stopped before Future completed.
```

It happend at my 2nd launch of the app.

**Comment by Insoleet** [comment]
Should be fixed in 0.32.3
